app/src/main/res/values-en-rGB/strings.xml

```diff
@@ -0,0 +1,894 @@
+<resources>
```

Comment:
Shouldn't the directory be named `values-en-GB` rather than `values-en-rGB`?

Reply:
@caugner I can't remember the exact research I did at the time, but I tested it out and I worked, and this I found just now seems to validate my previous thinking:

https://stackoverflow.com/a/23885951/2501304

What do you reckon?

Reply:
Sorry, you're absolutely right. I'll correct my review.